doc/quickstart/actinia_quickstart.rst

```diff
@@ -120,22 +54,8 @@ The option ``--persistent MAPSET_NAME`` allows the execution of commands
 in the **persistent** user database. It can be used with
 ``--location LOCATION_NAME``\ option.
 
-Requirements and setup
-~~~~~~~~~~~~~~~~~~~~~~~
-
-Needed Python libraries
-^^^^^^^^^^^^^^^^^^^^^^^
-
-In case not yet present on the system, the following Python libraries
```

Comment:
Can we assume the presence of this on the OSGeo Live DVD/usb? If so, we can remove this section.

Reply:
The quickstart should work for the OSGeoLive.